code/src/main/java/com/onlinecontacttracing/servlets/authentication/CheckForApiAuthorizationServlet.java

```diff
@@ -75,8 +76,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       TokenResponse tokenResponse = flow.newTokenRequest(code).setRedirectUri(url + getServletURIName()).execute();
 
       Credential credential = flow.createAndStoreCredential(tokenResponse, userId);
+      
       updateUser(userId, email);
       useCredential(userId, credential, response);
+      response.getWriter().println(email);
```

Comment:
Do we actually want this, or is it for debugging?

Reply:
Removed now